app/models/user.rb

```diff
@@ -355,15 +355,19 @@ def vet360_contact_info
     @vet360_contact_info ||= VAProfileRedis::ContactInformation.for_user(self)
   end
 
+  def va_profile_email
+    vet360_contact_info&.email&.email_address
+  end
+
   def all_emails
-    vet360_email =
+    the_va_profile_email =
```

Comment:
I don't feel like this rescue is necessary. Thoughts?

Reply:
if a user isn't initialized in va profile then it will raise an error when you try to fetch their va profile contact info

Reply:
What throws the error in this case? is it the `VAProfileRedis::ContactInformation` class? Wondering if we should implement the error handling there, and then we'll be able to safely call `vet30_contact_info`

Reply:
https://github.com/department-of-veterans-affairs/vets-api/blob/master/lib/va_profile/contact_information/service.rb#L155
this is where the error is raised. the error handling would be a separate issue but you would need to look into whether or not it's a good idea to suppress all errors every time `VAProfileRedis::ContactInformation` is used